src/config.py

```diff
@@ -13,6 +13,13 @@ class Config:
     MONGODB_PORT = 27017
     SUBSTRATE = "k8s"
     CONTAINER_NAME = "mongos"
+    USER_ROLE_CREATE_USERS = "admin"
+    SERVICE_NAME = "mongod"  # this must match the name of the service in the ROCK
```

Comment:
Shouldn't this be `mongos` and not `mongod` as we run `mongos`?
```suggestion
    SERVICE_NAME = "mongos"  # this must match the name of the service in the ROCK
```

Reply:
No, I think its something with how we package the ROCK. I had tried this earlier and it failed